.github/workflows/CI.yml

```diff
@@ -117,6 +117,10 @@ jobs:
         id: get-latest-tag
       - name: e2e test
         run: make e2e-test
+        env:
+          GIT_LATEST_TAG: ${{ steps.get-latest-tag.outputs.tag }}          
+      - name: e2e doc test
+        run: make doc-test
```

Comment:
Could you please make it parallel?

Reply:
Putting the doc test in another job, instead of an action after e2e.

Reply:
Sure